packages/AlgernonTests.package/ALGTestTypes.class/instance/testOpenSubmenuEntries.st

```diff
@@ -1,12 +1,16 @@
 testing docking bar types
 testOpenSubmenuEntries
 
-	|flatWidgetLookName fontSizeName |
+	| flatWidgetLookName fontSizeName newFontSizeName |
 	
 	flatWidgetLookName := 'Flat Widget Look'.
 	self runQuery: flatWidgetLookName.
 	self assert: (self resultsIncludeName: flatWidgetLookName).
 	
 	fontSizeName := 'Increase Font Size'.
+	"Item name changed in squeak v6.0"
+	newFontSizeName := 'increase font size (+)'.
 	self runQuery: fontSizeName.
-	self assert: (self resultsIncludeName: fontSizeName).
\ No newline at end of file
+	self assert: (
+		(self resultsIncludeName: fontSizeName) or: 
+		[self resultsIncludeName: newFontSizeName]).
```

Comment:
@Kryptolyser, ist da das "or" wichtig oder kann man das alte (`fontSizeName`) auch einfach weglassen?

Reply:
No, the `or` is important, because we have different submenu entries depending on the squeak version. If you would remove the old `fontSizeName` it would fail on Squeak 5.3 CI.